packages/react-microlink/src/components/Card/CardWrap.js

```diff
@@ -46,6 +46,10 @@ const style = css`
 
   ${({cardSize}) => isLarge(cardSize) && largeStyle}
 
+  ${({cardSize, reverse}) => reverse && css`
+    flex-direction: ${isLarge(cardSize) ? 'column-reverse' : 'row-reverse'}
```

Comment:
What do you think into move this line into a variable, similar to `largeStyle` ? Just for make the code more readable